src/components/MapComponents/MapLayer.js

```diff
@@ -54,8 +54,10 @@ export default {
      
       if (this.before && map.getLayer(this.before)) {
         map.addLayer(this.options, this.before)
-      } else {
+      } else if (map.getLayer('gl-draw-polygon-fill-inactive.cold')) {
```

Comment:
Any initial layers get added before the draw controller, so this layer is not yet on the map